### ISSUE TITLE
Highlight the whole Widget Area when it is selected

### ISSUE THREAD

**Issue by mapk** [WordPress/gutenberg#26465]
### Problem
When clicking a collapsed widget area, the area's heading is highlighted with a blue border. This is good. However when the area is opened, the heading remains highlighted instead of the whole area. This doesn't look right.

**Example of problem**
![highlight](https://user-images.githubusercontent.com/617986/97213011-5ab40f00-177e-11eb-8bbd-afb72c898b12.gif)

### Solution
Let's make the whole widget area highlighted in a blue border when it is selected, not just the area heading.

![Screen Shot 2020-10-26 at 11 28 38 AM](https://user-images.githubusercontent.com/617986/97213075-6dc6df00-177e-11eb-8156-7c9628f89087.png)

**Comment by talldan** [comment]
I closed a similar issue related to this. I don't think this is the right thing to do, because it'd involve removing a focus style from the expand/collapse button, which is basically an accessibility regression.

None of the other accordions in WordPress do this, so I'm not sure why the widget area should be different.
